modules/aioseop_feature_manager.php

```diff
@@ -27,16 +27,19 @@ function __construct( $mod ) {
 			parent::__construct();
 			$this->module_info = array(
 				'sitemap'           => array(
-					/* translators: the XML Sitemaps module allows users to generate a sitemap in .xml format for their website and
+					/*
```

Comment:
This looks incorrect if done this way for wrapping a comment on a new line. Could probably use...

```
/*
 * translators: comment.
 */
```

Reply:
grunt tried its best to make sense of a wrongly formatted comment but things such as this that are totally wrong may need manual intervention. not a breaking change though as it is just a comment

Reply:
Ah ok, so this was grunt and not you?

Normally it would be a 1 line comment, but this was a bit different with the length. I checked to see what WPCS considered valid, and it seems to asks for...

```
/*
 * translators: comment.
 */
```

With an empty line preceding it. I tried this change, and the old version, both styles get flagged by WPCS. Should it be corrected now, or on the next WPCS?